timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -226,35 +216,56 @@ def get_preset_models(
             logger.log(20, f"Custom Model Type Detected: {model}")
             model_type = model
 
-        model_type_kwargs = dict(
-            path=path,
-            freq=freq,
-            prediction_length=prediction_length,
-            eval_metric=eval_metric,
-            eval_metric_seasonal_period=eval_metric_seasonal_period,
-            hyperparameters=model_hps,
-            **kwargs,
-        )
+        model_hps_list = hyperparameters[model]
+        if not isinstance(model_hps_list, list):
+            model_hps_list = [model_hps_list]
+
+        for model_hps in model_hps_list:
+            ag_args = model_hps.pop(ag.constants.AG_ARGS, {})
+            for key in ag_args:
+                if key not in VALID_AG_ARGS_KEYS:
+                    logger.warning(f"WARNING: Unknown ag_args key: {key}")
```

Comment:
Shall we raise error instead of warning to avoid unintended behavior from original expectation?

Reply:
Agreed, changed this to `ValueError`.